dev/com.ibm.ws.security.common.jsonwebkey/test/com/ibm/ws/security/common/jwk/impl/JwKRetrieverTest.java

```diff
@@ -148,10 +150,26 @@ public void testGetPublicKeyFromJwk_publicKeyLocationInvalid() throws Exception
         JwKRetriever jwkRetriever = new JwKRetriever(configId, sslConfigurationName, jwkEndpointUrl,
                 jwkSet, sslSupport, hnvEnabled, null, null, publickey, keyLocation);
 
-        PublicKey publicKey = jwkRetriever.getPublicKeyFromJwk(kid, null);
+        PublicKey publicKey = jwkRetriever.getPublicKeyFromJwk(kid, null, true);
 
         assertNull("There must not be a public key.", publicKey);
     }
+    
+  
+    // check that when useSystemPropertiesForHttpClientConnections is passed in, client gets created with correct option
+    @Test
+    public void testGetPublicKeyFromJwk_useSystemProperties() throws Exception {
+        keyLocation = "badKeyLocation";
+        String jwkEndpointUrl2 = "http://somewheretotallybogusurl";
+        MockJwKRetriever jwkRetriever = new MockJwKRetriever(configId, sslConfigurationName, jwkEndpointUrl2,
+                jwkSet, sslSupport, hnvEnabled, null, null, publickey, keyLocation);        
+        try{
+            PublicKey publicKey = jwkRetriever.getPublicKeyFromJwk(kid, null, true);
+        } catch (Exception re){
+            // ioe expected
```

Comment:
Is there any kind of string we can verify in the exception message to be sure we got the exception we expected?